projects/schematics/src/migrations/migrations.json

```diff
@@ -50,6 +50,11 @@
       "version": "4.0.0",
       "factory": "./4_0/removed-public-api-deprecations/removed-public-api-deprecation#migrate",
       "description": "Comment about usage of removed public api"
+    },
+    "migration-v4-rename-symbol-01": {
```

Comment:
We're probably going to change this soon, but let's keep it aligned now - `migration-v4-rename-symbol-02`